main.py

```diff
@@ -76,118 +84,52 @@ async def on_ready(self):
         if not self.synced:
             await tree.sync(guild=discord.Object(id=aloraID))
             self.synced = True
-        logger.warning(f"Debug mode is on, remember about it")
+
+        # try:
+        #     cursor.execute("CREATE")
+        # except Error as e:
+        #     logger.error(f"{e}")
+        #     return None
+
+
+
+        if debugMode == True: logger.warning(f"Debug mode is on, remember about it")
         logger.info(f"We have logged in as {self.user}")
         logger.info(f"Servers:")
         for idx, guild in enumerate(self.guilds):
             logger.info(f"{idx+1}. {guild.name}({guild.id})")
-            # start_xp = 1000
-            # sum = start_xp
-            # idx = 1
+            for member in guild.members:
+                try:
+                    cursor.execute(f"UPDATE members SET voice_join_time = %s WHERE id = %s", (datetime.now(), member.id))
+                except Error as e:
+                    logger.error(f"{e}")
+                else:
+                    mysqldb.commit()
+
+            # for doc in db.collection("levels").get():
+            #     levels_dict = db.collection("levels").document(doc.id).get().to_dict() 
+            #     try:
+            #         cursor.execute("INSERT INTO levels (role_id, required_points) VALUES (%s, %s)", (levels_dict["role_id"], math.floor(levels_dict["required_points"])))
+            #     except Error as e:
+            #         logger.error(f"{e}")
+            #         return None
+            #     else:
+            #         mysqldb.commit()
```

Comment:
If you have some ideas and you're afraid you'll forget about it, spoiler, you won't (ONLY IF YOU'LL CREATE ISSUES AND PROPER PLANNING FOR YOUR PROJECT). Just delete the code if it doesn't work and write it again in separate branch created for this specific feature.